php/Plugin.php

```diff
@@ -88,11 +89,77 @@ public function require_deprecated() {
 		}
 	}
 
+	/**
+	 * Require the API functions.
+	 */
+	private function require_api() {
+		require_once __DIR__ . '/BlockApi.php';
+	}
+
 	/**
 	 * Requires helper functions.
 	 */
 	private function require_helpers() {
-		require_once __DIR__ . '/Helpers.php';
-		require_once __DIR__ . '/BlockApi.php';
+		if ( $this->is_plugin_conflict() ) {
+			add_action( 'admin_notices', [ $this, 'plugin_conflict_notice' ] );
+		} else {
+			require_once __DIR__ . '/Helpers.php';
+		}
+	}
+
+	/**
+	 * Gets whether there is a conflict from another plugin having the same functions.
+	 *
+	 * @return bool Whether there is a conflict.
+	 */
+	public function is_plugin_conflict() {
+		return function_exists( 'block_field' ) || function_exists( 'block_value' );
+	}
+
+	/**
+	 * An admin notice for another plugin being active.
+	 */
+	public function plugin_conflict_notice() {
```

Comment:
Most of this file is [taken from](https://github.com/getblocklab/block-lab/pull/696/files#diff-f162295a163d95e718f07b67b57ff1e1R101) Block Lab's upcoming release.